.github/workflows/PR.yml

```diff
@@ -26,10 +26,11 @@ jobs:
           poetry install
       - name: Run linters
         run: |
-          poetry run ni-python-styleguide lint ./generated/
-          poetry run ni-python-styleguide lint ./nidaqmx_examples/
-          poetry run ni-python-styleguide lint ./src/
-          poetry run ni-python-styleguide lint ./tests/
+          poetry run ni-python-styleguide lint ./src/codegen/properties
+          poetry run ni-python-styleguide lint ./src/codegen/utilities
+          poetry run ni-python-styleguide lint ./src/codegen/__init__.py
+          poetry run ni-python-styleguide lint ./src/codegen/__main__.py
+          poetry run ni-python-styleguide lint ./src/codegen/generator.py
```

Comment:
Why not lint all of ./src/codegen? Is there something in ./src/codegen that you are trying to avoid linting?

Reply:
This question is still relevant

Reply:
You can exclude the metadata:
`poetry run ni-python-styleguide lint ./src/codegen --exclude ./src/codegen/metadata`

Reply:
better yet, put this in the pyproj file:
```toml
[tool.ni-python-styleguide]
extend_exclude = "./src/codegen/metadata"
```
that way, dev's don't have to use the param and can just "poetry run ni-python-styleguide lint"

Reply:
Try this exclude list:
```
[tool.ni-python-styleguide]
extend_exclude = "docs,generated/nidaqmx,src/nidaqmx,src/codegen/metadata"
```

You will need to use `black` to reformat a few files in `tests/` before `ni-python-styleguide lint` will pass with this exclude list.